grid/project-management/tasks/profile.supr

```diff
@@ -181,74 +205,120 @@ usecase UpdateTask {
 }
 
 """
-Delete a task in a project
+Delete Task
+Delete a task in project management systems.
 """
-usecase DeleteTask {
+usecase DeleteTask unsafe {
```

Comment:
Good point, semantically it should be `idempotent`.

Reply:
Although, on second thought - I can obviously handle a `404` for an already deleted resource (making it truly idempotent), but what if you end up passing a wrong `taskId`, and end up mistakenly thinking you've deleted the task because there's no error? :thinking: